app/src/components/main/scrollWrapper/scrollWrapper.scss

```diff
@@ -37,4 +37,6 @@
   transition: opacity 400ms
 }
 .scrolling-content {
+  display: flex;
+  flex-direction: column;
```

Comment:
These changes breaks another components which are using scrollWrapper component. Do we really need to change scrollWrapper styles?